test/parallel/test-crypto-aes-wrap.js

```diff
@@ -1,62 +1,69 @@
 'use strict';
 const common = require('../common');
-if (!common.hasCrypto)
-  common.skip('missing crypto');
+const { describe, it } = require('node:test');
 
-const assert = require('assert');
-const crypto = require('crypto');
+// Disabling the unnecessary lint rule here. We're testing the common.hasCrypto
+// in the skip option of the describe block.
+/* eslint-disable node-core/crypto-check */
```

Comment:
I wonder why don't we just skip crypto on the python runner? This would avoid spawning those threads?

Reply:
Likely worthwhile.